src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -170,7 +170,9 @@ def battery_pool(
 
         if key not in self._battery_pools:
             self._battery_pools[key] = BatteryPool(
-                batteries_status_receiver=self._battery_status_channel.new_receiver(),
+                batteries_status_receiver=self._battery_status_channel.new_receiver(
+                    maxsize=1
+                ),
```

Comment:
I guess this is done because it only needs to keep the latest update, right? Otherwise please clarify it in the commit message of this change

Reply:
it is mentioned in the BatteryPool documentation: https://github.com/frequenz-floss/frequenz-sdk-python/blob/c565ddf9e2e240c55e2873eb351daf4589e262db/src/frequenz/sdk/timeseries/battery_pool/battery_pool.py#L42-L43

Just realized we're doing only one of those.  I will do the other one also in the same commit.

Reply:
updated.